sql/12dbSpringBreakCampForm.sql

```diff
@@ -0,0 +1,17 @@
+-- Table for the dbSpringBreakCampForm
+CREATE TABLE dbSpringBreakCampForm (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    email TEXT NOT NULL,
+    student_name VARCHAR(256) NOT NULL,
+    school_choice TEXT NOT NULL,
+    isAttending BOOLEAN NOT NULL,
+    waiver_completed BOOLEAN NOT NULL,
+    notes TEXT,
+    
+    family_id INT,
```

Comment:
I could be wrong, but I believe this should be child_id instead of family_id since this form is for students part of the brain builders program.